backend/src/test/java/de/trailmate/backend/controller/TourControllerTest.java

```diff
@@ -84,29 +85,9 @@ void whenAddTourisCalled_isTourAddedCorrectlyAndReturnAddedTour() throws Excepti
                         content(jsonObj))
                 .andExpect(MockMvcResultMatchers.status()
                         .isOk())
-                .andExpect(MockMvcResultMatchers
-                        .content().json(jsonObj));
+                .andExpect(jsonPath("$.title").value("DGHDsgdhsdg"));
```

Comment:
```suggestion
                .andExpect(MockMvcResultMatchers
                        .content().json(jsonObjWithoutId))
                        .andExpect(jsonPath("$.id").isPresent());
```

or something like this...